### ISSUE TITLE
Basemap Style for tileserver-gl

### ISSUE THREAD

**Issue by geo-tech** [syncpoint/openvtpk#8]
Could you provide your modifications for using Basemap Style in tileserver-gl?

**Comment by ThomasHalwax** [comment]
Of course, I will create a new github page where I am going to share the styles.

**Comment by ThomasHalwax** [comment]
I have updated the documentation to style the [basemap.at](https://github.com/syncpoint/openvtpk#local-basemapat-vector-tiles-with-styles-applied)

**Comment by geo-tech** [comment]
Thank you for sharing the documentation to style the basemap. 

Do you also have the following erros in the actually style?
layers[566]: duplicate layer id "GIP_OUTSIDE_BAUWERK_L_BRÜCKE/112111/1
layers[570]: duplicate layer id "GIP_OUTSIDE_BAUWERK_L_BRÜCKE/112111/0
layers[722].layout.text-rotate: missing required property "stops""
layers[723].layout.text-rotate: missing required property "stops""
layers[724].layout.text-rotate: missing required property "stops""
layers[725].layout.text-rotate: missing required property "stops""
layers[726].layout.text-rotate: missing required property "stops""
layers[727].layout.text-rotate: missing required property "stops""

**Comment by ThomasHalwax** [comment]
You are right, the styles shipped by basemap.at are creating lots of errors. I've transferred the documentation and a corrected version of the ```root.json``` style file to a new repository: [basemap.at](https://github.com/syncpoint/basemap.at). Can you please verify that the doc and the styles are correct?
Thank you!
